user.rb

```diff
@@ -1,16 +1,20 @@
 require "pry"
 
 class User
-  @@all_users = []
+  @all_users = []
   attr_accessor :first_name, :last_name, :email
 
   def self.all
-    @@all_users
+    @all_users
+  end
+
+  def self.all=(array_of_users)
```

Comment:
Nice 👍. Rails has the concept of a `cattr_accessor`, which does something similar (but I think ultimately still uses class variables).